### ISSUE TITLE
Issue with the website giving error screen

### ISSUE THREAD

**Issue by FelipeBarrosCode** [DefangLabs/defang#704]
![image](https://github.com/user-attachments/assets/62306088-49aa-4fb3-91b6-b11673ef1a31)

![image](https://github.com/user-attachments/assets/3623242c-3946-470b-a894-80b17c6ba88b)

The website on Chrome is giving me an error and making me unable to use the playground. I suspect that this could be something related to section management with cookies because I saw that Google might be restricting the use of third-party cookies. In addition, it could also be something related to the connection to the backend, considering that it warns about the connection closing early. There are more error messages shown on the pictures above ^.

**Comment by ve-varun-sharma** [comment]
Perhaps posting some relevant code snippets related to the problem may help trouble shoot the issue here!

In this case potentially posting code related to this backend connection component you mentioned would be good!

**Comment by FelipeBarrosCode** [comment]
Yes, it would, but I do not have access to the code. The problem was in the Defang website, not mine. I did not find anywhere else to post about this, so I posted here. The only code to which I have access is the HTML, CSS and JS on Google developer tools.

**Comment by ve-varun-sharma** [comment]
> Yes, it would, but I do not have access to the code. The problem was in the Defang website, not mine. I did not find anywhere else to post about this, so I posted here. The only code to which I have access is the HTML, CSS and JS on Google developer tools.

Ahhh Okay I understand now, I thought you were referring to your own site hosted via Defang, but it's the actual defang site.

okay thanks for clarifying and posting this!
